net/i2pd/Makefile

```diff
@@ -9,20 +10,20 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=i2pd
-PKG_VERSION:=2.38.0
+
+PKG_VERSION:=2.40.0
+
 PKG_RELEASE:=1
+PKG_BUILD_PARALLEL:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=https://codeload.github.com/PurpleI2P/i2pd/tar.gz/$(PKG_VERSION)?
-PKG_HASH:=8452f5323795a1846d554096c08fffe5ac35897867b93a5079605df8f80a3089
+PKG_HASH:=4443f484ad40753e892170a26c8ee8126e8338bf416d04eab0c55c1c94a4e193
 
 PKG_MAINTAINER:=David Yang <mmyangfl@gmail.com>
 PKG_LICENSE:=BSD-3-Clause
 PKG_LICENSE_FILES:=LICENSE
 
-PKG_BUILD_PARALLEL:=1
-PKG_INSTALL:=1
```

Comment:
Where is went?

Reply:
Returned.

Reply:
But really, why we need it? We don't have `install` target (and not plannig adding of it).

Reply:
I was asking you, why it was removed, because it was nowhere documented or said in commit or pull request. If it is not needed anymore, it can be removed. ;)